### PR TITLE
Added Winter-2016-300

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
I've added the first list of our norm data collection to Concepticon.
Please check carefully whether I've included all necessary files and information, as this is my first contribution to conception.

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -276,3 +276,4 @@ Nordlinger-2019-57	Nordlinger, Rachel and Thieberger, Nick	2019		57	basic	Englis
...
 Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list of 100 items is based on the [Leipzig-Jakarta concept list](:ref:Tadmor-2009-100) and was employed in the study of phonosemantic patterns in the basic vocabulary of 66 genealogically unrelated languages.	1-12	
 Tolmie-1884-211	Tolmie, William F. and Dawson, George M.	1884		211	basic	English	Languages of the Native Tribes of British Columbia	https://open.library.ubc.ca/collections/bcbooks/items/1.0222394	Tolmie1884	Tolmie1884	This collection of 211 items, based on [Instructions for Research Relative to the Ethnology and Philology of America](:bib:Gibbs1863), was collected by the authors during expeditions in the British Columbia territory during the winters of 1875 and 1876. It contains basic vocabulary terms in addition to local flora and fauna terms. The core list of 211 items, also known as Dawson-Tolmie Vocabularies, is extended by appendices with extended vocabularies for specific languages in the area, but is not considered part of the main vocabulary list.		Tolmie-Dawson Vocabulary
+Winter-2016-300	Bodo Winter	2016		300	ranked	English
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: Of course, we need a longer explanation here, and also a link to the bibtex.

**Comment by AnnikaTjuka** [comment]
> @AnnikaTjuka, did you run `concepticon test`? Did id fail?
> 
No, I din't run `concepticon test` but just tested it and got an Error warning (`inconsistent data in repository`). After including the Bibtex link in conceptlists.tsv everything was OK.

> @AnnikaTjuka, the only thing needed is to add a bit more info in concepticon.tsv. Otherwise, this looks fine.
> 
I've also included a short description in the conceptlists.tsv

**Comment by AnnikaTjuka** [comment]
> One more thing: if there's a link to Winter's data on github, could you add teh URL in conceplists.tsv?

I already did that in the last commit but forgot to mention it. I've included [https://github.com/bodowinter/iconicity_senses/tree/master/data](https://github.com/bodowinter/iconicity_senses/tree/master/data) in the URL column.

**Comment by LinguList** [comment]
good, somehow, I didn't see it, my fault.

**Comment by LinguList** [comment]
So I suggest, we wait for one more reviewer, and if this is seen as okay, we can merge.

**Comment by chrzyki** [comment]
Looks good!

**Pull request merged**